### ISSUE TITLE
Inconsistent with the paper (Stage-II inputs)

### ISSUE THREAD

**Issue by abhineet99** [hanzhanggit/StackGAN-Pytorch#26]
In models.py, it looks like the Stage 2 Generator takes a text-embedding and noise as input. In the paper, it also takes stage 1 Generator output. 
In trainer, one can see that there is no difference in the stage1 and stage2 training, as they both take only text embedding input. 
From paper- "The Stage-II GAN takes Stage-I results and text descriptions as inputs".
Let me know if I have misinterpreted anything.

**Comment by abhineet99** [comment]
Mistake at my end. It takes as input text embedding and noise only, but calls stage-1 GAN inside its forward function.
`    def forward(self, text_embedding, noise):
        _, stage1_img, _, _ = self.STAGE1_G(text_embedding, noise)
        stage1_img = stage1_img.detach()
        encoded_img = self.encoder(stage1_img)
`
Closing the issue.
